Products/Zuul/facades/triggersfacade.py

```diff
@@ -719,15 +761,22 @@ def importTriggers(self, triggers):
         """
         Add any new trigger definitions to the system.
 
-        Note: modifies the triggers argument to add 'new_uuid' to the definition.
+        Note: modifies the triggers argument
+              to add 'new_uuid' to the definition.
 
         Does not attempt to link a trigger to a notification.
         """
         existingTriggers = [x['name'] for x in self.getTriggerList()]
-        existingUsers = {"{} (User)".format(x.id): IGlobalIdentifier(x).getGUID() for x in self._dmd.ZenUsers.getAllUserSettings()}
-        existingUsers.update({"{} (Group)".format(x.id): IGlobalIdentifier(x).getGUID() for x in self._dmd.ZenUsers.getAllGroupSettings()})
+        existingUsers = {
+            "{} (User)".format(x.id): IGlobalIdentifier(x).getGUID()
+            for x in self._dmd.ZenUsers.getAllUserSettings()
+        }
+        existingUsers.update({
+            "{} (Group)".format(x.id): IGlobalIdentifier(x).getGUID()
+            for x in self._dmd.ZenUsers.getAllGroupSettings()}
+        )
```

Comment:
Have the closing curly brace inline with the closing parenthesis to improve the symmetry of the code, e.g.
```
existingUsers.update({
    ...
})
```